cve_bin_tool/checkers/bluez.py

```diff
@@ -13,9 +13,9 @@ def get_version(lines, filename):
 
        VPkg: bluez, bluez
     """
-    regex = [r"bluetoothctl: ([5]+\.[0-9]+\.[0-9]+)", r"bluetoothctl: ([5]+\.[0-9]+)"]
+    regex = [r"bluetoothctl: ([5]+\.[0-9]+\.[0-9]+)", r"([5]+\.[0-9]+)"]
```

Comment:
Whoops, merged this and then had to back it out because I noticed something: this change to the checker is going to cause false positives because 5.* is definitely a string that's going to show up in other places.

Reply:
@terriko I have confirmed that by scanning a real bluetoohctl file. The pattern bluetoothctl: 5.*.* Doesn't exist in the Ubuntu version of bluetoothctl and it showed no other pattern which we  could possibly use. But I want to point that we are calling regex only after confirming that file is bluetoothctl. Although we can say that it will produce a false positive but the probability of finding a file named bluetoohctl and then the file containing string 5.* is really low. 

Anyway I'll look to improve this checker.

Reply:
Oh, goodness, that's incorrect.  The liklihood of any file on linux (including bluetoothctl) containing a string 5.X is actually pretty high, because strings usually include things like glibc and gcc, both of which have ever-evolving versions that could include the number 5.  It's basically only a matter of time before a 5.X string will make a false positive.  And then it gets worse worse, because I fixed the checker in #442 so it'll detect things that merely "contain" bluez, so you're going to be potentially finding matches on any file.

Reply:
For example:

```console
(venv3.6) terri@terrioda-desk2:~/tmp/openssh$ strings usr/bin/ssh-keygen | grep -i glib
GLIBC_2.2.5
GLIBC_2.14
GLIBC_2.4
GLIBC_2.25
GLIBC_2.8
GLIBC_2.3
GLIBC_2.26
GLIBC_2.3.4
GLIBC_2.17
```

Reply:
@terriko I'll see if can come up with some solution. Thanks for the review.

Reply:
@terriko I think the bluez checker needs to be updated. There are two reasons for that
1) I can't find any string having expression as ```bluetoothctl: 5.X.X```. All the version strings including the old version(bluez 3, 4) are having versions as "3.X" and "4.X"
more read: [http://www.bluez.org/download/](url)

2) version 2, 3 contained libs and utils package
 
Terriko, I'm still working on this checker.